.travis.yml

```diff
@@ -0,0 +1,47 @@
+language: php
+sudo: false
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+
+matrix:
+    fast_finish: true
+    include:
+          # Minimum supported PHP and Symfony version
+        - php: 5.5
+          env: DEPENDENCIES="minimum"
+
+          # Test the latest stable release
+        - php: 5.5
+        - php: 5.6
+        - php: 7.0
+        - php: 7.1
+        - php: 7.2
+
+          # Test LTS versions
+        - php: 7.2
+          env: DEPENDENCIES="symfony/lts:v2"
+        - php: 7.2
+          env: DEPENDENCIES="symfony/lts:v3"
+
+          # Latest beta release
+        - php: 7.2
+          env: DEPENDENCIES="beta"
+
+    allow_failures:
+          # Latest beta is allowed to fail.
+        - php: 7.2
+          env: DEPENDENCIES="beta"
```

Comment:
As short as the beta phases are, I'd prefer not to have the last beta fail. Dev yes, beta should be stable IMO.